Sources/protoc-gen-swift/FileIo.swift

```diff
@@ -46,36 +46,8 @@ class Stderr {
 
 class Stdout {
   static func print(_ s: String) { printToFd(s, fd: 1) }
-  static func write(bytes: Data) {
-    bytes.withUnsafeBytes { (p: UnsafePointer<UInt8>) -> () in
-      _ = _write(1, p, bytes.count)
```

Comment:
Can we also delete the `_write` definition above?

Reply:
👍 Done.